cvat-ui/src/reducers/tasks-reducer.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2020 Intel Corporation
+// Copyright (C) 2021 Intel Corporation
```

Comment:
The right option is 2020-2021.
We should keep initial year also according to internal rules.